Source/WebCore/Modules/url-pattern/URLPattern.h

```diff
@@ -54,9 +54,9 @@ class URLPattern final : public RefCounted<URLPattern> {
     static ExceptionOr<Ref<URLPattern>> create(ScriptExecutionContext&, std::optional<URLPatternInput>&&, URLPatternOptions&&);
     ~URLPattern();
 
-    ExceptionOr<bool> test(ScriptExecutionContext&, std::optional<URLPatternInput>&&, String&& baseURL) const;
+    ExceptionOr<bool> test(ScriptExecutionContext&, std::optional<URLPatternInput>&&, std::optional<String>&& baseURL) const;
```

Comment:
This doesn't look right, neither does the parameter to `match()`. And optional String on IDL side becomes a null string (`s == String()` where `s.isNull()` returns true), not std::nullopt. Using std::optional for Strings in WebKit doesn't make sense since they have a null state.